contrib/roles/linux/ovn-kubernetes/vars/ubuntu.yml

```diff
@@ -7,9 +7,11 @@ ovn_kubernetes_info:
   branch: master
 
 ovn_kubernetes_binaries:
-  - ovnkube
-  - ovn-k8s-overlay
-  - ovn-k8s-cni-overlay
-  - ovn-kube-util
-  - ovnkube.exe
-  - ovn-k8s-cni-overlay.exe
+  linux:
+    - ovnkube
+    - ovn-k8s-overlay
```

Comment:
This one is gone now. Was not used anyway directly.

Reply:
Updated the PR to reflect the changes.